src/haz3lweb/view/Page.re

```diff
@@ -143,7 +143,12 @@ module Update = {
       model |> return_quiet;
     | FinishImportAll(None) => model |> return_quiet
     | FinishImportAll(Some(data)) =>
-      Export.import_all(~import_log, data, ~specs=ExerciseSettings.exercises);
+      Export.import_all(
+        ~import_log,
+        data,
+        ~specs=ExerciseSettings.exercises,
+        ~tutorial_specs=TutorialSettings.exercises,
```

Comment:
let's call Tutorial pages "lessons" to distinguish them from Exercise mode's "exercises"